### DISCUSSION TITLE
Advanced search options

### DISCUSSION THREAD

**Discussion by Robroi2000** [ezra-bible-app/ezra-bible-app#343] [Ideas]
It would be helpful to be able to use advanced search options like wildcard characters, logical operators, Regex, searching within selected books or verse lists, etc.

**Comment by tobias-klein** [comment]
Thanks for your proposal! Adding more search options has also been on my mind since some time. The one option that I think would be most useful as a next step would be to search within groups of books. I imagine the following list to choose from:

* OT
* Pentateuch
* Maybe other groups of books from the OT
* NT
* Gospels

Are you already familiar with the current tab search - enabled by CTRL + f?

Other options (wildcard, advanced logic, etc.) are not on my priority list, but we can surely consider those in the future!

**Reply by Robroi2000** [reply]
I already discovered the tab search, but there seems to be a bug. As soon as I click into the verse list, the search does not work anymore.

**Reply by tobias-klein** [reply]
The tab search has been under re-work for the next release 1.1. I'll have another look and try to ensure that this bug is not in the next release, which is going to be published about three weeks from now!
